pkg/sql/alter_type.go

```diff
@@ -112,13 +113,17 @@ func (n *alterTypeNode) startExec(params runParams) error {
 		}
 		eventLogDone = true // done inside alterTypeOwner().
 	case *tree.AlterTypeDropValue:
-		if params.p.SessionData().SafeUpdates {
-			err = pgerror.DangerousStatementf(
-				"DROP VALUE may cause view/default/computed expressions to stop working if the " +
-					"enum value is used inside them")
-		} else {
-			err = params.p.dropEnumValue(params.ctx, n.desc, t.Val)
+		if !params.p.SessionData().DropEnumValueEnabled {
+			return pgerror.WithCandidateCode(
+				errors.WithHint(
+					errors.WithIssueLink(
+						errors.New("ALTER TYPE ... DROP VALUE ... is only supported as an alpha feature"),
```

Comment:
nit: change to "only supported as an experimental feature"

i think "alpha" isn't a standard term in this context.